src/components/DayPickerRangeController.jsx

```diff
@@ -1111,6 +1110,42 @@ export default class DayPickerRangeController extends React.PureComponent {
     return { currentMonth, visibleDays };
   }
 
+  getCachedPropByDay(day, prop, props, forceUpdate = false) {
```

Comment:
is there a reason this is cached on the instance, instead of being done in react state?